src/styles/variables.css

```diff
@@ -0,0 +1,6 @@
+:root {
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --button-color: #00acdc;
+  --button-additional-color: #fff;
```

Comment:
describe a color or `card-button-color` for example

Reply:
Ok.